### DISCUSSION TITLE
[RFC] Form support names group

### DISCUSSION THREAD

**Discussion by elevensky** [ant-design/ant-design#49961] [Ideas]
## Summary
能不能给 Form.Item name 增加，group分组功能
<Form.Item name="step1" group="one">

form.getFieldsValueByGroup(groupName: string | string[])

## Motivation
场景：一个大表单拆三步，第一步含有20个字段，如果想下一步要校验这20个字段的话现阶段要单个列出来，有分组之后可以通过分组来归类某些name,做批量的获取值或者校验

**Comment by crazyair** [comment]
实现成只加载当前激活的 Step 表单字段，也只会校验加载的字段，没加载的不会校验。

**Reply by elevensky** [reply]
那这个举例不太恰当(不过可能也有一些情况需要预渲染出来)，不过有这种对某些字段整体归类处理会方便很多
